app/main.py

```diff
@@ -1 +1,53 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
```

Comment:
Skills should be object attribute, not class attribute. It should be created in init. Fix everywhere